src/main/java/com/raisetecch/handling/demo/UserController.java

```diff
@@ -20,11 +26,19 @@ public UserController(UserService userService) {
         this.userService = userService;
     }
 
-    @GetMapping("/users/{id}")
+    @GetMapping("/users/{id}")//dbの中のユーザーの検索
     public User getUser(@PathVariable("id") int id) {
         return userService.findUser(id);
     }
 
+    @PostMapping("/entry")//dbにユーザーの登録
+    public User postUser(@RequestBody User user) {
+        if (user.getName() == null || user.getName().isEmpty()) {
+            throw new ResourceNotFoundException("Name must not be empty or null.");
+        }
+        return userService.entryUser(user.getId(), user.getName());
```

Comment:
idをinputにする必要はないかと思います！
```suggestion
        return userService.entryUser(user.getName());
```